### PR TITLE
Add CONVECTIVE HEAT FLUX GAUGE output

### PR DESCRIPTION
**Pull request by johodges**
Similar output to radiometer for radiation. For use with validation studies which have split gauge measured heat fluxes into convective and radiative components.

### CODE REVIEW

File: Source/dump.f90 [hunk 1]
```diff
@@ -9242,7 +9242,14 @@ REAL(EB) FUNCTION SOLID_PHASE_OUTPUT(NM,INDX,Y_INDEX,Z_INDEX,PART_INDEX,OPT_WALL
          ENDDO SH_MATL_LOOP
          SOLID_PHASE_OUTPUT = SOLID_PHASE_OUTPUT * 0.001_EB
       ENDIF
-
+   CASE(76) ! CONVECTIVE HEAT FLUX GAUGE
+         IF (PY%HEAT_TRANSFER_COEFFICIENT>=0._EB) THEN
```
Comment (rmcdermo) [inline Source/dump.f90:9246]: More tab issues?  Please set you editor to use spaces only.

Comment (johodges) [reply Source/dump.f90:9246]: My editor is set up for spaces only, but when I committed the updates there were some pre-existing tabs which my editor replaced with spaces. I reverted those back to the tabs which were already there thinking they were intentional. Do you want me to commit with those converted to spaces?

Comment (rmcdermo) [reply Source/dump.f90:9246]: Yes, please go ahead and correct the tabs to spaces issue (per developer guidelines, only spaces should be used).  Thanks

**Comment by mcgratta** [comment]
What file had the pre-existing tabs?

**Comment by rmcdermo** [comment]
> What file had the pre-existing tabs?

Apparently, dump.f90

**Comment by mcgratta** [comment]
I cannot find any tabs in dump.f90

**Comment by rmcdermo** [comment]
@johodges I was only worried about dump.  That is where my comment was.  Not in the user guide.

**Comment by mcgratta** [comment]
There is only one place where we actually need tabs, and this is the makefile. Elsewhere, tabs are a menace.

**Comment by johodges** [comment]
The tabs were only in the user guide .tex file. I do not see any in dump either.

**Pull request merged**